scargo/cli.py

```diff
@@ -404,12 +404,13 @@ def run(
 @cli.command()
 def test(
     verbose: bool = Option(False, "--verbose", "-v", help="Verbose mode."),
+    profile: str = Option("Debug", "--profile", help="CMake profile to use"),
```

Comment:
Is there a use case for using a different profile than Debug?

Reply:
Not really, but users can override this to release at least during migration period - I noticed some tests in projects (mainly using asserts) that were incorrectly passing so far. It could be useful to narrow down such issues.